src/test/resources/com/scriptbasic/testprograms/TestSelect.bas

```diff
@@ -0,0 +1,43 @@
+'
+' This program is used in unit testing the jScriptBasic interpreter to test the functionality of the
```

Comment:
Create a test case in this program that executes a "CASE" and the selected branch calls a subroutine that also has a "CASE" command, which selects one branch, and another case when the subroutine "CASE" does not select any of the branches and returns unselected.